gobblin-runtime/src/test/java/org/apache/gobblin/runtime/api/MysqlMultiActiveLeaseArbiterTest.java

```diff
@@ -35,10 +35,10 @@
 
 @Slf4j
 public class MysqlMultiActiveLeaseArbiterTest {
-  private static final int EPSILON = 10000;
-  private static final int MORE_THAN_EPSILON = (int) (EPSILON * 1.1);
-  private static final int LINGER = 50000;
-  private static final int MORE_THAN_LINGER = (int) (LINGER * 1.1);
+  private static final long EPSILON = 10000;
```

Comment:
ideally it should be 10000L , longs should end in "L" or "l", let me finish review, if this is the only thing, you can do it in next PR